src/Committee/CommitteeManager.php

```diff
@@ -457,7 +457,7 @@ public function changePrivilege(Adherent $adherent, Committee $committee, string
         }
     }
 
-    public function getCoordinatorCommittees(Adherent $coordinator, CommitteeFilters &$filters): array
+    public function getCoordinatorCommittees(Adherent $coordinator, CommitteeFilter $filters): array
```

Comment:
Maybe you can change the name of variable too ?

Reply:
it's an oversight, done 👍